lib/scout_apm/payload/metadata.ex

```diff
@@ -11,7 +11,7 @@ defmodule ScoutApm.Payload.Metadata do
 
   def new(timestamp) do
     %__MODULE__{
-      app_root: System.cwd(),
+      app_root: File.cwd(),
```

Comment:
System.cwd returns a string or nil, but File.cwd returns a tuple of `{:ok, string}` or `{:error, reason}` so this will need to be changed to account for that